src/App.tsx

```diff
@@ -1,8 +1,15 @@
+import React from "react";
 import Calendar from "./components/Calendar"
+import SettingsContext from "./contexts/SettingsContext";
 
 function App() {
+	const [dob, setDOB] = React.useState(new Date());
+	const [lifeExpectancy, setLifeExpectancy] = React.useState(76);
```

Comment:
just curious, in the future would this come from the DB?

Reply:
Yes, this will come from DB.